notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
<p>"Aggregating Data" would be a more appropriate title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I do not want to stress a specific function in that section and use the aggregation just as an example of applying _methods_ in general. As such, I use _do something_ in order to explain the general concept of methods applied on DataFrame/Series.</p>